### PR TITLE
[XPipeline]Camera UI - Unify Headers - Output

### PR DESCRIPTION
**Pull request by alex-vazquez**
---
### Purpose of this PR
Refactoring Camera UI code for Output
image

This PR also fixes this https://fogbugz.unity3d.com/f/cases/1322297/

---
### Testing status
Check that the properties are properly edited

---
### Comments to reviewers
Another step for URP/ HDRP camera unification

Allow dynamic resolution should be on Output, there is already an entry on the Changelog with the Camera Editor reorder, that's why I didn't add one in this PR.

### CODE REVIEW

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Output.Drawers.cs [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+        partial class Output
+        {
+            public static readonly CED.IDrawer Drawer = CED.FoldoutGroup(
+                Styles.header,
+                Expandable.Output,
+                k_ExpandedState,
+                FoldoutOption.Indent,
+                CED.Group(
+#if ENABLE_VR && ENABLE_XR_MANAGEMENT
+                Drawer_SectionXRRendering,
```
Comment (jenniferd-unity) [inline com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Output.Drawers.cs:21]: should this one be indented once more?

**Review by jenniferd-unity — APPROVED** [review]

**Review by RSlysz — APPROVED** [review]

File: com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Output.Drawers.cs (hunk 1 above)
Comment (alex-vazquez) [reply com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Output.Drawers.cs:21]: I will check :)

**Comment by alex-vazquez** [comment]
@Unity-Technologies/gfx-qa-hdrp , could you do a quick smoke test on this section of the camera editor? Thank you ;)

**Review by TomasKiniulis — APPROVED** [review]
Approving. Checked all Output properties with multiple cameras and multiple Game views. Not encountering any PR specific issues

**Comment by sebastienlagarde** [comment]
@alex-vazquez : need to merge with master + caution formatting job don't pass

**Pull request merged**